20151222/rails-5.0.0.beta1-demo/test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @post = posts(:one)
+    @comment = comments(:one)
+  end
+
+  test "should get index" do
+    get comments_url(post_id: @post)
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_comment_url(post_id: @post)
+    assert_response :success
+  end
+
+  test "should create comment" do
+    assert_difference('Comment.count') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.